CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 2.0.12 - in dev
 
 ### New features
+ - new directive : luid table grid
```

Comment:
I'm gonna need more than that, see release [2.0.0](https://github.com/LuccaSA/lucca-ui/releases/tag/2.0.0), there's an entire section dedicated to the then new directive `luid-user-picker`

also, there is no documentation on how to use it and the deno page hasn't been updated